ENGLISH.md

```diff
@@ -13,68 +13,99 @@ two members of our organization:
 This website was originally made to learn web development in Ruby on Rails.
 However, the source code keeps on expanding as we add on more features.
 
-## Setup with Vagrant
-Note: the Vagrant setting provided uses 2 GB of RAM. If you feel that it's
-a lot, edit Vagrantfile; change `vb.memory` from `"2048"` to `"1024"` or
-however you want.
+## Initial Setup
+Note: the following set-up procedure runs on Ubuntu 20.04 LTS. These steps are
+recommended although need not be followed exactly. If you wish to run
+Ubuntu on another OS, you may want to consider
+[VirtualBox](https://www.virtualbox.org/wiki/Downloads) to run Ubuntu in a
+virtual machine. 
+
+### Prerequisite packages
+For the project to run on your machine, you would need to install several packages:
```

Comment:
Wrong numbering?

Also, can write the commands for Ubuntu? `sudo apt install software-properties-common` etc.

Reply:
Edited